src/context/NotesContext/NotesContext.jsx

```diff
@@ -37,7 +37,27 @@ const NotesProvider = ({ children }) => {
 			},
 		});
 	};
-	const updateNotes = () => {};
+	const updateNotes = (_id, note) => {
+		fetchNotes({
+			method: 'POST',
+			url: `/api/notes/${_id}`,
+			headers: {
+				authorization: localStorage.getItem('token'),
```

Comment:
Shouldnt you first be checking if the token exists or not for possible edge cases?

Reply:
The token will not empty as it is being generate at the very start without auth functionality.